webapp/src/ts/modules/reports/reports.component.ts

```diff
@@ -97,13 +98,11 @@ export class ReportsComponent implements OnInit, AfterViewInit, OnDestroy {
     this.tourService.startIfNeeded(this.route.snapshot);
     this.setActionBarData();
 
-    this.currentLevel = this.sessionService.isOnlineOnly() ? Promise.resolve() : this.getCurrentLineageLevel();
+    this.currentLevel = this.authService.online(true) ? Promise.resolve() : this.getCurrentLineageLevel();
```

Comment:
Could we define `ngOnInit` as async and `await this.getCurrentLineageLevel()` here directly?  
It would make it easier to read here and in `prepareReports()` below.

If that's possible, you would need to add `await`s in front of `component.ngOnInit()` calls in the tests in `webapp/tests/karma/ts/modules/reports/reports.component.spec.ts`

Reply:
I'm confident of doing `this.sessionService.isOnlineOnly()` to `this.authService.online(true)` because it is essentially the same. 
This other change makes a lot of sense, but I'm not 100% confident about doing it in this PR. I wonder if there's a reason behind like to prevent a race condition or to not block something for too long, I need to test the breadcrumbs.
 
Is it okay if we do it in another PR to not mix features & testing?

Reply:
Yes it makes sense, let's keep it separate from this PR